src/Sylius/Behat/Context/Ui/EmailContext.php

```diff
@@ -70,6 +70,22 @@ public function aWelcomingEmailShouldHaveBeenSentTo($recipient)
     }
 
     /**
+     * @Then a verification email should have been sent to :recipient
+     */
+    public function aVerificationEmailShouldHaveBeenSentTo($recipient)
+    {
+        $this->assertEmailContainsMessageTo('To verify your email address', $recipient);
+    }
+
+    /**
+     * @Then a verification email should not have been sent to :recipient
+     */
+    public function aVerificationEmailShouldNotHaveBeenSentTo($recipient)
+    {
+        $this->assertEmailNotContainsMessageTo('To verify your email address', $recipient);
+    }
```

Comment:
These steps should be unified with the scenarios [here](https://github.com/michalmarcinkowski/Sylius/blob/0ce1d75d208bdbd265199d04f5c7939318d94f21/features/account/verification/verifying_email_address.feature).